contracts/strategies/RFPStrategy.sol

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.19;
+
+import {BaseStrategy} from "./BaseStrategy.sol";
+import {Metadata} from "../core/libraries/Metadata.sol";
+import {ReentrancyGuard} from "@openzeppelin/security/ReentrancyGuard.sol";
+
+contract RFPStrategy is BaseStrategy, ReentrancyGuard {
+    /// @notice Struct to hold details of an recipient
+    struct Recipient {
+        address payoutAddress;
+        uint256 proposedAmount;
```

Comment:
Do we need the proposed amount? Wouldn't the entire pool go to the one recipient?

Reply:
Yes, it's part of the spec.
It's named different in the spec, but here is the link: https://github.com/allo-protocol/allo-v2/blob/dd1b6b137b44f780717b4228d107e384b90fa2ad/contracts/strategies/allocation/rfp-simple/README.md?plain=1#L23